packages/server/src/course/course.entity.ts

```diff
@@ -113,13 +98,13 @@ export class CourseModel extends BaseEntity {
   @DeleteDateColumn()
   deletedAt?: Date;
 
-  @ManyToOne(
-    type => OrganizationCourseModel,
-    organizationCourse => organizationCourse.course,
+  @OneToOne(
+    (type) => OrganizationCourseModel,
+    (organizationCourse) => organizationCourse.course,
   )
-  @Exclude()
   organizationCourse: OrganizationCourseModel;
 
   @Column({ nullable: true })
+  @Exclude()
   organizationCourseId: number;
```

Comment:
I don't think we need this column if we alreay hve organizationCourse relationship above